src/main/java/org/spongepowered/api/world/chunk/ProtoChunk.java

```diff
@@ -98,49 +104,22 @@
     Vector3i chunkPosition();
 
     /**
-     * Gets the containing {@link ProtoWorld} of this {@link ProtoChunk}. As
-     * it may vary based on the status of both {@link Game} and {@link Server},
-     * this {@link ProtoChunk} may be likewise used for world generation, in
-     * which case, the {@link ProtoWorld} would not be a {@link World} instance.
+     * Gets the {@link Ticks number of ticks} players have been present in this
+     * chunk, used for calculation of the regional difficulty factor. In vanilla,
+     * it is increased by the number of players in the chunk every tick, and is
+     * capped at 3,600,000 ticks (50 hours).
      *
-     * <p>It can be inferred however, that if {@link #state()} returns
-     * {@link ChunkStates#WORLD_READY}, the {@link ProtoWorld} would be a
-     * {@link World} instance. Inversely, if {@link #state()} returns
-     * {@link ChunkStates#EMPTY}, the {@link ProtoWorld} would not be a
-     * valid {@link World} object.</p>
-     *
-     * @return The parented world
+     * @return The number of ticks
      */
-    ProtoWorld<?> world();
```

Comment:
What happened to the reference to the world?

Reply:
Why shouldn't we be able to soft guarantee that any chunk has a world? Or was it that generating chunks may not actually have the links to the generation regions?

Reply:
Generating chunks (ProtoChunks) have no link to thier target world or anything that links to something like a `GenerationRegion`.